### PR TITLE
BLD: removal of pascal interface

### PR DESCRIPTION
**Pull request by mbechtold**
This PR includes
- the removal of all Pascal interface files, and  
- the related cleanup of the Makefile.
If there is interest in the use of this interface at a later stage, this PR can be reversed.

### CODE REVIEW

**Review by lbusschaert — APPROVED** [review]
Nice! It's much cleaner without all the interfaces and Pascal units. Will be much better for the users.

**Review by gdelannoy — CHANGES_REQUESTED** [review]
Looks great! To be consistent, also remove the "optional build instructions" from the introduction page?

**Comment by mbechtold** [comment]
> Looks great! To be consistent, also remove the "optional build instructions" from the introduction page?

**Comment by mbechtold** [comment]
Thanks both. Good point @gdelannoy: I removed the related build instructions.

**Review by gdelannoy — APPROVED** [review]
Looks good! See also PR Louise: I'd rephrase to "AquaCrop v7.0 and higher versions" and remove the version number from the itemised list of what is distributed to generalise the README file.
